scene/gui/code_edit.cpp

```diff
@@ -479,19 +479,10 @@ void CodeEdit::gui_input(const Ref<InputEvent> &p_gui_input) {
 			accept_event();
 			return;
 		}
-		if (k->is_action("ui_home", true)) {
-			code_completion_current_selected = 0;
-			code_completion_force_item_center = -1;
-			queue_redraw();
-			accept_event();
-			return;
-		}
-		if (k->is_action("ui_end", true)) {
-			code_completion_current_selected = code_completion_options.size() - 1;
-			code_completion_force_item_center = -1;
-			queue_redraw();
-			accept_event();
-			return;
+		if (
+				k->is_action("ui_text_caret_line_start", true) ||
+				k->is_action("ui_text_caret_line_end", true)) {
```

Comment:
I think it's more readable on a single line than on three like this (I know it's clang-format doing it but it makes it not worth it).

Reply:
That's a left over of these being 4 checks instead of 2. I'm correcting this.